verilog/formatting/tree_unwrapper.cc

```diff
@@ -1321,6 +1376,49 @@ void TreeUnwrapper::ReshapeTokenPartitions(
       break;
     }
 
+    case NodeEnum::kRandomizeMethodCallExtension:
+    case NodeEnum::kRandomizeFunctionCall: {
+      auto& last = *ABSL_DIE_IF_NULL(partition.RightmostDescendant());
+      if (PartitionIsCloseParen(last) || PartitionStartsWithSemicolon(last)) {
+        verible::MergeLeafIntoPreviousLeaf(&last);
+      }
+
+      auto& uwline = partition.Value();
+      const auto partition_policy = uwline.PartitionPolicy();
+      if (partition_policy == PartitionPolicyEnum::kAppendFittingSubPartitions) {
+        if (partition.Children().size() > 2) {
```

Comment:
Can you explain this with a comment?

Reply:
It was fix for randomize function with constraints. Moved it here https://github.com/google/verible/pull/246/files#diff-7872d97b05fbc6f2a64cf14b8ae74d75R863.

Btw. how the ```kWithConstraints``` should be handled? Now it looks like this:
```
  { ([<auto>], policy: fit-else-expand) @{19}, (origin: "function f;...endfunction")
    { ([function f ;], policy: fit-else-expand, (origin: "function f;")) }
    { (>>[<auto>], policy: always-expand) @{19,1}, (origin: "s = std::ra...comment2
};")
      { (>>[s = std::randomize (], policy: fit-else-expand) }
      { (>>[<auto>], policy: always-expand) @{19,1,1}, (origin: "aaa,bbb,ccc,ddd,eee")
        { (>>[aaa ,], policy: fit-else-expand, (origin: "aaa")) }
        { (>>[bbb ,], policy: fit-else-expand, (origin: "bbb")) }
        { (>>[ccc ,], policy: fit-else-expand, (origin: "ccc")) }
        { (>>[ddd ,], policy: fit-else-expand, (origin: "ddd")) }
        { (>>[eee], policy: fit-else-expand, (origin: "eee")) }
      }
      { (>>[) with {], policy: uninitialized) }
      { (>>>>[<auto>], policy: always-expand) @{19,1,3}, (origin: "a == e;")
        { (>>>>[// comment1], policy: fit-else-expand) }
        { (>>>>[a == e ;], policy: fit-else-expand, (origin: "a == e;")) }
        { (>>>>[// comment2], policy: fit-else-expand) }
      }
      { (>>[} ;], policy: fit-else-expand) }
    }
    { ([endfunction], policy: always-expand, (origin: "endfunction")) }
  }
```
Should it look like ```if () begin end``` (with the ```with``` keyword appended to last argument?)

Reply:
This looks good with the `) with {` partition in the middle.  (No need to attach it to the `eee` that precedes it.)